Cargo.toml

```diff
@@ -1,10 +1,13 @@
 [package]
 name = "pyver"
-description = "Python PEP-440 Version Parsing"
-authors = ["Allstreamer <allstreamer.contact@gmail.com>"]
+description = "Python PEP-440 Version Parser"
+authors = [
+    "Allstreamer <allstreamer.contact@gmail.com>",
+    "Jan Bronicki <janbronicki@gmail.com>",
+]
 license = "MIT"
 version = "0.1.0"
```

Comment:
did you bump the crate version?

Reply:
No versions have been bumped yet still gotta do that but I ran out of time for today

Reply:
ok

Reply:
Versions have been bumped